### ISSUE TITLE
Consider renaming tools-python to spdx-tools-python

### ISSUE THREAD

**Issue by ssbarnea** [spdx/tools-python#767]
The fact that the python package does not match the repository name and that the "tools-python" as a name does not say anything is causing confusions.

I already made the big mistake of cloning and *installing* another repository that happened to be named 'spdx-tools'... Basically the unfortunate decision to not use the same name on repository and python package name, will make victims.

While for someone working on `spdx` full time, that is implied but for almost everyone else 'tools-python' does say nothing.

**Comment by maxhbr** [comment]
yeah, that is a confusing situation. We are discussing how to make it clearer.
